Fitness/Controllers/HomeController.swift

```diff
@@ -269,7 +297,8 @@ extension HomeController: UICollectionViewDelegate, UICollectionViewDelegateFlow
     }
 
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        if collectionView != mainCollectionView {
+        print("here")
```

Comment:
Nitpicky but remove prints for PR's!

Reply:
Thanks for letting me know, will keep in mind for future PRs